tests/publiccloud/azure_cli.pm

```diff
@@ -27,7 +27,8 @@ sub run {
         add_suseconnect_product 'sle-module-public-cloud';
         zypper_call('in azure-cli jq python3-susepubliccloudinfo');
     }
-    assert_script_run('az --version');
+    record_soft_failure('bsc#1192671') if (script_run('az --version', die => 0) != 0);
+    record_soft_failure('bsc#1192671') if (script_run('az --help', die => 0) != 0);
```

Comment:
in case you wonder why I'm adding `--help` is because it also returns !=0
https://bugzilla.suse.com/show_bug.cgi?id=1105223